Author: Jasper Grimm <jasper.grimm@york.ac.uk>
Description: define _GNU_SOURCE if not already defined, so that compilation succeeds on some older OS versions
diff -Nru qt-everywhere-src-6.11.2.p3/qtwebengine/src/3rdparty/chromium/third_party/libdrm/src/xf86drm.c qt-everywhere-src-6.11.2.p4/qtwebengine/src/3rdparty/chromium/third_party/libdrm/src/xf86drm.c
--- qt-everywhere-src-6.11.2.p3/qtwebengine/src/3rdparty/chromium/third_party/libdrm/src/xf86drm.c	2026-08-27 14:03:22.164297058 +0100
+++ qt-everywhere-src-6.11.2.p4/qtwebengine/src/3rdparty/chromium/third_party/libdrm/src/xf86drm.c	2026-08-27 14:04:48.011118132 +0100
@@ -31,6 +31,10 @@
  * DEALINGS IN THE SOFTWARE.
  */
 
+#ifndef _GNU_SOURCE
+#define _GNU_SOURCE
+#endif
+
 #include <stdio.h>
 #include <stdlib.h>
 #include <stdbool.h>
